src/OctoshiftCLI.Tests/Octoshift/Services/SecretScanningAlertServiceTests.cs

```diff
@@ -358,15 +369,86 @@ public async Task Migrates_Multiple_Alerts()
             TARGET_REPO,
             100,
             SecretScanningAlert.AlertStateResolved,
-            SecretScanningAlert.ResolutionRevoked)
+            SecretScanningAlert.ResolutionRevoked,
+            "Revokation explanation comment 1")
         );
 
         _mockTargetGithubApi.Verify(m => m.UpdateSecretScanningAlert(
             TARGET_ORG,
             TARGET_REPO,
             300,
             SecretScanningAlert.AlertStateResolved,
-            SecretScanningAlert.ResolutionFalsePositive)
+            SecretScanningAlert.ResolutionFalsePositive,
+            "False positive explanation comment")
+        );
+    }
+
+    [Fact]
+    public async Task One_Secret_Updated_With_No_Resolution_Comment()
```

Comment:
Not sure if this adds value, seems like a bit of an overkill IMO.

Reply:
What would be your preferred option:
1. remove completely that test. This would align with the current tests for code scanning where a dismissedComment is always set (so we are not testing the use case where no comments were set)
2. rework both code scanning and secret scanning tests to "squeeze" one alert with no comment (to exercise the code, but not as a separate test case).
3. keep that test separate and add a similar overkill test in code scanning 😄

Reply:
So I am leaning towards option 1, since the way I think about this is a scenario where no comments are set still means we're setting it but to a `null` value and we have test coverage that shows we indeed use that value.